### PR TITLE
Added testing for Python 3.10.

### PR DESCRIPTION
**Pull request by PetrDlouhy**

### CODE REVIEW

**Review by felixxm — COMMENTED** [review]
@PetrDlouhy Thanks for this patch :+1: `"3.10"` should be added to the test workflow (`.github/workflows/test.yml`). Please add add also a release note, e.g.
```
* Confirmed support for Python 3.10.
```
and classifier:
```
'Programming Language :: Python :: 3.10',
```

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```
Comment (felixxm) [inline .github/workflows/test.yml:10]:
```suggestion
        python-version: [3.6, 3.7, 3.8, 3.9, "3.10"]
```

Comment (PetrDlouhy) [reply .github/workflows/test.yml]: Yeah, that is a catch :-) Fixed.

**Comment by felixxm** [comment]
@PetrDlouhy Thanks :+1:

**Pull request merged**